tests/test_bot.py

```diff
@@ -116,6 +116,22 @@ def test_detect_self_bump(self, _) -> None:
 
         self.cleanup()
 
+    def test_detect_url(self, _) -> None:
+        """Test the ability of KarmnaBot to detect a URL which contains either the ++ or
```

Comment:
We do not test for the `++` tokens here. There's a comment above that `++` cannot be exposed bare in a URL so we either need to update the docstring or our add a `++` test case.